test/services/publisher_test.rb

```diff
@@ -115,12 +115,6 @@ def test_publish_should_raise_exception_if_topic_is_nil
       end
     end
 
-    def test_publish_should_raise_exception_if_topic_is_nil
```

Comment:
@dougal Why did you remove this test?

Reply:
It's a duplicate.

https://github.com/iHiD/propono/blob/master/test/services/publisher_test.rb#L112-L122

Reply:
Wonderful. Thanks.